src/main/java/com/paneedah/mwc/weapons/AuxiliaryAttachments.java

```diff
@@ -1627,10 +1627,10 @@ public static void init(Object mod) {
                 .withName("APC9Action").withRenderablePart()
                 .withTextureName("Dummy.png").build(ModernWarfareMod.MOD_CONTEXT);
         
-        SA80Action = new AttachmentBuilder<Weapon>().withCategory(AttachmentCategory.EXTRA2)
-                .withModel(new com.paneedah.mwc.models.SA80Action(), "sa80.png")
-                .withName("SA80Action").withRenderablePart()
-                .withTextureName("Dummy.png").build(ModernWarfareMod.MOD_CONTEXT);
+//        SA80Action = new AttachmentBuilder<Weapon>().withCategory(AttachmentCategory.EXTRA2)
+//                .withModel(new com.paneedah.mwc.models.SA80Action(), "sa80.png")
+//                .withName("SA80Action").withRenderablePart()
+//                .withTextureName("Dummy.png").build(ModernWarfareMod.MOD_CONTEXT);
```

Comment:
It is preferable to just delete this, anyway thanks to GitHub if we ever need it we can just see it in the history.